src/app/components/CustomIcons/ExplorerIcon.tsx

```diff
@@ -1,23 +1,31 @@
 import { FC } from 'react'
 import SvgIcon, { SvgIconProps } from '@mui/material/SvgIcon'
 
+export const viewBoxAspectRatio = 782.02 / 146.19
```

Comment:
This is a little tricky to see in code. But I ok I guess.

Reply:
yeah maybe it;s not worth doing. I will bring back hardcoded value